pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1809,6 +1809,35 @@ public class Foo {
         </example>
     </rule>
 
+
+    <rule name="UseDiamondOperator"
+          language="java"
+          since="6.11.0"
+          message="Explicit type arguments can be replaced by Diamond Operator "
+          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_codestyle.html#usediamondoperator"
+          minimumLanguageVersion="1.7">
+        <description>Use the diamond operator to let the type be inferred</description>
```

Comment:
Maybe we can expand a bit on the description?

With the Diamond operator it is possible to avoid duplication of the type parameters. Instead, the compiler is now able to infer the parameter types for constructor calls, which makes the code also more readable.